index.js

```diff
@@ -220,6 +220,13 @@ function onStream (opts, client, onerror) {
       next()
     })
 
+    if (opts.payloadLogFile) {
+      if (!client._payloadLogFile) {
+        client._payloadLogFile = require('fs').createWriteStream(opts.payloadLogFile, {flags: 'a'})
+      }
+      pump(stream, client._payloadLogFile)
```

Comment:
@Qard hmm `pump` might actually close the file once `stream` ends. I'll just investigate if this works for more than one request

Reply:
Fixed